frontend/src/components/dashboard/varianceHomePage/VarianceHomePage.js

```diff
@@ -68,67 +66,69 @@ const propTypes = {
   history: PropTypes.shape({ push: PropTypes.func }).isRequired,
 };
 
-const joinOrRemove = (param, key) => (isEmpty(param) ? {} : { [key]: param.join(",") });
-const formatParams = ({ region = [], compliance_code = [], ...remainingParams }) => ({
-  ...joinOrRemove(region, "region"),
-  ...joinOrRemove(compliance_code, "compliance_code"),
-  ...remainingParams,
-});
-
+export const joinOrRemove = (param, key) => {
+  if (isEmpty(param)) {
+    return {};
+  }
+  return typeof param === "string" ? { [key]: param } : { [key]: param.join(",") };
+};
+export const removeEmptyStings = (param, key) => (isEmpty(param) ? {} : { [key]: param });
+export const formatParams = ({
+  region = [],
+  compliance_code = [],
+  issue_date_after,
+  issue_date_before,
+  expiry_date_before,
+  expiry_date_after,
+  search,
+  major,
+  ...remainingParams
+}) => {
+  return {
+    ...joinOrRemove(region, "region"),
+    ...joinOrRemove(compliance_code, "compliance_code"),
+    ...removeEmptyStings(issue_date_after, "issue_date_after"),
+    ...removeEmptyStings(issue_date_before, "issue_date_before"),
+    ...removeEmptyStings(expiry_date_before, "expiry_date_before"),
+    ...removeEmptyStings(expiry_date_after, "expiry_date_after"),
+    ...removeEmptyStings(search, "search"),
+    ...removeEmptyStings(major, "major"),
```

Comment:
What makes these different from the `remainingParams` that causes them to need to be handled by `removeEmptyStrings`?

Reply:
There was some edge case where a url param 'expiry_date_after=' was getting pushed up.  This keeps the param from being pushed up